File: backend-api/Models/Ward.cs
using System;
using System.Collections.Generic;

namespace CareFlowAI.API.Models
{
    public class Ward
    {
        public Guid Id { get; set; } = Guid.NewGuid();
        public string WardNumber { get; set; } = string.Empty;
        public string WardType { get; set; } = string.Empty; // e.g., ICU, General
        public int Capacity { get; set; }
        public int OccupiedBeds { get; set; }
    }
}

File: mobile_app/lib/models/Ward.dart
class Ward {
  final String id;
  final String wardNumber;
  final String wardType;
  final int capacity;
  final int occupiedBeds;

  Ward({
    required this.id,
    required this.wardNumber,
    required this.wardType,
    required this.capacity,
    required this.occupiedBeds,
  });

  factory Ward.fromJson(Map<String, dynamic> json) {
    return Ward(
      id: json['id'] ?? '',
      wardNumber: json['wardNumber'] ?? '',
      wardType: json['wardType'] ?? '',
      capacity: json['capacity'] ?? 0,
      occupiedBeds: json['occupiedBeds'] ?? 0,
    );
  }
}